File: 0-9/2.py
# -*- coding:utf-8 -*-
# 题目: 替换空格


class Solution:
    # s 源字符串
    def replaceSpace(self, s):
        # write code here
        result = ''
        for index in s:
            if index == ' ':
                result += '%20'
            else:
                result += index
        return result


obj = Solution()
print(obj.replaceSpace('We Are Happy'))

File: 0-9/2.go
package main

import "fmt"

// 题目: 替换空格

// ReplaceSpace ...
func ReplaceSpace(s string) string {
	//write your code here
	sNew := ""
	for _, str := range s {
		if " " == string(str) {
			sNew += "%20"
		} else {
			sNew += string(str)
		}
	}
	return sNew
}

func main() {
	fmt.Println(ReplaceSpace("We Are Happy."))
}
